src/mkdocstrings/handlers/__init__.py

```diff
@@ -212,7 +212,12 @@ def __init__(self, collector: BaseCollector, renderer: BaseRenderer) -> None:
         self.renderer = renderer
 
 
-def get_handler(name: str, theme: str, custom_templates: Optional[str] = None) -> BaseHandler:
+def get_handler(
+    name: str,
+    theme: str,
+    custom_templates: Optional[str] = None,
+    get_handler_config: Optional[Callable[[str], dict]] = None,
```

Comment:
See above comment, would be `handler_config` (dict), or even `**handler_config`.